src/hooks/useSystemIntakeContacts.tsx

```diff
@@ -45,31 +51,46 @@ function useSystemIntakeContacts(
   // Get system intake from Apollo cache
   const { systemIntake } = useSystemIntake(systemIntakeId);
```

Comment:
I'm not really sure of the necessity of the custom hook for system intakes.  Apollo by default should cache the data.  I know this was existing outside of the scope of verify users, but just wanted to mention this.

Reply:
Thanks for pointing this out! I was initially using the hook for something else and ended up forgetting to get rid of it. Definitely doesn't make sense to have a custom hook just to get the intake values in one place.